Server/Components/Pawn/Scripting/Player/Natives.cpp

```diff
@@ -292,6 +298,11 @@ SCRIPT_API(TogglePlayerClock, bool(IPlayer& player, bool enable))
 	return true;
 }
 
+SCRIPT_API(PlayerHasClockEnabled, bool(IPlayer& player))
```

Comment:
i suggest that function name can be changed to **IsPlayerClockEnabled**

Reply:
The names are lifted straight from old includes.